esp/platform/txsummary.cpp

```diff
@@ -663,3 +684,67 @@ CTxSummary::TxEntryBase* CTxSummary::queryEntry(const char* key)
 
     return nullptr;
 }
+
+StringBuffer& CTxOpenTelemetryConnector::normalizeKey(const char* txKey, const char* otKey, StringBuffer& normalized) const
```

Comment:
txKey is originalKeyName, and otkey is alternateKeyName, right?
If so, let's use the most selfdocumenting names when possible.